docs/imviz/displayimages.rst

```diff
@@ -424,55 +459,15 @@ To set the percentile for all the images at once:
     plot_options.stretch_vmin = 0
     plot_options.stretch_vmax = 1000
 
-Colormap
---------
-
-The spectrum of colors used to visualize data can be changed using this drop-down.
-
-From the API
-^^^^^^^^^^^^
-
-To set the colormap for just the image being displayed:
-
-.. code-block:: python
-
-    viewer = imviz.default_viewer
-    viewer.colormap_options
-    viewer.set_colormap('Viridis')
-
-To set the colormap for all the images at once:
-
-.. code-block:: python
-
-    plot_options = imviz.plugins['Plot Options']
-    plot_options.select_all()
-    plot_options.image_colormap = 'Viridis'
-
-.. _imviz_custom_colormap:
-
-Adding Custom Colormap
-^^^^^^^^^^^^^^^^^^^^^^
-
-A custom colormap can only be added when Imviz is run in a notebook, not from the
-command line. The custom colormap must be added to Glue *before* starting Imviz.
-The example below adds a random colormap generated by ``photutils`` into Glue:
-
-.. code-block:: python
-
-    from glue.config import colormaps
-    from photutils.utils import make_random_cmap
-
-    randcmap = make_random_cmap(ncolors=256)
-    randcmap.colors[0] = 0.  # To make your background black
-    colormaps.add('photutils_cmap', randcmap)
-
-Only after the above is done can you start Imviz and use the custom colormap:
-
-.. code-block:: python
+Contour
+-------
 
-    imviz = Imviz()
-    imviz.load_data('myimage.fits')
-    imviz.default_viewer.set_colormap('photutils_cmap')
+This option selects whether to show or hide contours. It is off by default and can be
+turned on by clicking the eye icon. The :guilabel:`Contours` of a second image can
+be plotted over a first image or cube. The contours of the second image will appear
+superimposed on the first image. If contours are overplotted on a cube, the contours
+will remain unchanged as you scrub through the cube. Please note that this feature is in
+development and will be improved in the future.
```

Comment:
I think this language is consistent with actual behavior since you're mentioning _images_, but just a note that I do see contours of a _cube_ (correctly) update when slicing... do there need to be any updates to this to clarify?